### PR TITLE
<AutoCompleteWithLabel />, <InputWithLabel /> - style fix

### PR DESCRIPTION
**Pull request by saarkuriel**
### 🔦 Summary

According to this Jira:
https://jira.wixpress.com/browse/PREM-10763?filter=32448
The following style fixes should be applied.
Thanks

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX @wuwa via Anat Steiner 
- 📚 Change is documented
  - [x] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by mykas** [comment]
Hey from the JIRA ticket I see these requirements:
Screenshot 2020-02-17 at 13 51 34

what does it mean that content below should be pushed down? Maybe it meant to wrap when content is too long?

File: src/InputWithLabel/InputWithLabel.st.css [hunk 1]
```diff
@@ -46,10 +46,16 @@
...
 }
 
 .statusMessageIcon {
   margin-right: 5px;
   display: flex;
   margin-left: -2px;
+}
+
+.errorMessageContent {
+  line-height: 18px;
```
Comment (EgoziE) [inline src/InputWithLabel/InputWithLabel.st.css]: This is not standard and should be approved by wsr ux

File: src/AutoCompleteWithLabel/AutoCompleteWithLabel.scss [hunk 2]
```diff
@@ -11,10 +11,16 @@
...
 }
 
 .statusMessageIcon {
   margin-right: 5px;
   display: flex;
   margin-left: -2px;
+}
+
+.errorMessageContent {
+  line-height: 18px;
```
Comment (EgoziE) [inline src/AutoCompleteWithLabel/AutoCompleteWithLabel.scss]: This is not standard and should be approved by wsr ux

**Comment by saarkuriel** [comment]
> Hey from the JIRA ticket I see these requirements:
> Screenshot 2020-02-17 at 13 51 34
> 
> what does it mean that content below should be pushed down? Maybe it meant to wrap when content is too long?

That means that the error will not overflow if there's content below. That's sort of a given...

**Comment by mykas** [comment]
But then it just goes on and on
Screenshot 2020-02-17 at 13 55 57

Is that intentional?

**Comment by mykas** [comment]
> This PR should be reviewed by our UX

It seems that this component is not in our zeplin specs at all. I am not sure what is the status of these components. @nitayneeman @shlomitc can you tell us?

**Comment by shlomitc** [comment]
@mykas - this is a legitimate component in our design system, I asked our UX to add it to our sketch file.
cc- @milkyfruit

**Comment by saarkuriel** [comment]
> But then it just goes on and on
> Screenshot 2020-02-17 at 13 55 57
> 
> Is that intentional?
> It's not wrapping cause there are no spaces

What string will be that long without any spaces?

**Comment by mykas** [comment]
> > But then it just goes on and on
> > Screenshot 2020-02-17 at 13 55 57
> > Is that intentional?
> > It's not wrapping cause there are no spaces
> 
> What string will be that long without any spaces?

You're right this is not a good example.

**Comment by saarkuriel** [comment]
@mykas can we proceed with this? thanks

File: src/InputWithLabel/InputWithLabel.st.css (hunk 1 above)
Comment (mykas) [reply src/InputWithLabel/InputWithLabel.st.css]: @milkyfruit please check when you can

**Comment by milkyfruit** [comment]
If I'm correct this component was developed just for Premium, specs came from there too. There is no intention to promote this input to the rest of designers, so official specs don't exist too. Officially is promoted to use `<Input/>`.

To fix this component, Jira comments are correct. The component should behave like this:
- Component's height is auto
- Error message text should Wrap
- Component's bounding box should grow with wrapped text

**Comment by saarkuriel** [comment]
> If I'm correct this component was developed just for Premium, specs came from there too. There is no intention to promote this input to the rest of designers, so official specs don't exist too. Officially is promoted to use `<Input/>`.
> 
> To fix this component, Jira comments are correct. The component should behave like this:
> 
> * Component's height is auto
> * Error message text should Wrap
> * Component's bounding box should grow with wrapped text

If this is the situation, I'd be more than happy to take these components into premium. It would be much easier for us. @shlomitc please advise.

**Comment by shlomitc** [comment]
The component should stay part of `wix-style-react` and specs will be added in the near future.
At the moment, let's follow the guidelines as you receive

**Pull request merged**